src/sharepoint/information/INFORMATION.html

```diff
@@ -6,83 +6,69 @@
-
                 <div class="col-sm-6">
-                    <div class="oui-tile mb-5 h-100">
-                        <h3 class="oui-header_4 oui-tile__title" data-translate="sharepoint_dashboard_quota"></h3>
-                        <div class="oui-tile__body clearfix">
-                            <div class="col-md-6">
-                                <div id="diskUsed"
-                                     data-gauge-chart
-                                     data-chartdata="informationsCtrl.sharepoint.currentUsage/1024/1024/1024"
-                                     data-disk-unit="'sharepoint_dashboard_unit_size_giga' | translate"
-                                     data-chart-min="0"
-                                     data-chart-max="1024"
-                                     data-limit-warning="70"
-                                     data-limit-alert="85"
-                                     data-color-ok="#81BB28"
-                                     data-color-warning="#ECAC02"
-                                     data-color-alert="#E03937"
-                                     data-shadow-opacity="0.2">
-                                </div>
-                            </div>
-
-                            <div class="col-md-6">
-                                <dl>
-                                    <dt class="oui-tile__term" 
-                                        data-translate="sharepoint_dashboard_quota_total_label"></dt>
-                                    <dd class="mb-3"
-                                        data-ng-bind="informationsCtrl.sharepoint.quota | humanReadableSize"></dd>
-
-                                    <dt class="oui-tile__term"
-                                        data-translate="sharepoint_dashboard_quota_used_label"></dt>
-                                    <dd class="mb-3"
-                                        data-ng-bind="informationsCtrl.sharepoint.currentUsage | humanReadableSize"></dd>
-
-                                    <dt class="oui-tile__term"
-                                        data-translate="sharepoint_dashboard_quota_free_label"></dt>
-                                    <dd class="mb-3"
-                                        data-ng-bind="informationsCtrl.sharepoint.quota - informationsCtrl.sharepoint.currentUsage | humanReadableSize"></dd>
-                                </dl>
-                            </div>
-                        </div>
-                    </div>
+                    <oui-tile class="h-100" data-heading="{{:: 'sharepoint_dashboard_quota' | translate}}">
+                        <oui-tile-definition>
+                            <oui-tile-description>
+                                <oui-progress data-min-value="0"
+                                              data-max-value="1024">
+                                    <oui-progress-bar data-ng-if="informationsCtrl.sharepoint.currentUsage/1024/1024/1024 <= 70" 
```

Comment:
Ok I understand what you wanted to do :). What you suggested this morning felt easier :

```js
<oui-progress-bar data-type="{{ $ctrl.ouiProgressBarType }}" ...
```

Reply:
In fact, attribute type in oui-progress-bar is a one-time binding attribute, don't think we could use this tricks.